dotcom-rendering/scripts/env/gen-dotenv.js

```diff
@@ -0,0 +1,16 @@
+import { promises as fs } from 'fs';
+import * as path from 'path'
+import secrets from '../secrets';
+import { getGuardianConfiguration } from '../webpack/lib/aws-parameters';
+
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+(async () => {
+	const env = process.env.NODE_ENV === 'production' ? 'prod' : 'dev';
+  const configuration = await getGuardianConfiguration(env)
+	const envFile = path.resolve(__dirname, '../../.env');
+
+	for (const secretKey of secrets) {
+		// eslint-disable-next-line no-await-in-loop
+		await fs.appendFile(envFile, `${secretKey}=${configuration.getParameter(secretKey)}`)
+	}
+})()
```

Comment:
Is this wrapped in an IIFE only so it can be `async`?

- Could we use top-level `await`?
- Could we use the non-promise `fs` API?
- _🦕_ ? 😉

Reply:
The await was setup to avoid race conditions on writing lines to a file. We've removed the need for this.